boot/freeldr/freeldr/CMakeLists.txt

```diff
@@ -6,14 +6,6 @@ if(SEPARATE_DBG)
     set(CMAKE_LDR_PE_HELPER_STANDARD_LIBRARIES "-lgcc" CACHE STRING "Standard C Libraries")
 endif()
 
-if(MSVC)
-    # We don't need hotpatching
-    replace_compile_flags("/hotpatch" " ")
-
-    # Explicitly use string pooling
-    add_compile_flags("/GF")
```

Comment:
Can you put this back? (so that if the global ReactOS defaults change, freeldr will still be use string pooling)

Reply:
Why would freeldr need a particular treatment?

Reply:
Due to memory size and layout limitations.

Reply:
Done